ccloud/docs/beginner-cloud.rst

```diff
@@ -99,7 +93,7 @@ Create a new Confluent Cloud environment and specify it as the default
 
       ccloud environment create demo-script-env -o json
 
-#. Verify your output resembles:
+#. Verify your output resembles below. Note the value of the environment ID, in this case ``env-5qz2q``.
```

Comment:
```suggestion
Your output should resemble:. 
```

Reply:
With "note that..." , this would normally be formatted as an actual note, but since this occurs in several places, how about the approach taken here, and apply it throughout wherever there are similar instances?

Reply:
Generally, it's ok, but for this tutorial, since the ID values can vary, I think it's helpful to call out the part that may be different.  So I'll adjust this in my copy and push.

Reply:
Oh, I think I now understand your suggestion, which is to have `Your output should resemble` **before** the output and the "different" part (e.g. `In this case, the value of the environment ID is`...) **after** the output?  I don't have a strong opinion on this, just thought it's "cleaner" to have it on a single line, but we can switch it.  WDYT?